lib/brains/errors.py

```diff
@@ -21,6 +24,16 @@ def format(self):
                 'message': error.message
             })
 
+        cc_result = self.error.result.credit_card_verification
+        if cc_result:
+            log.debug('credit card processing error: {r}'.format(r=cc_result))
+            errors['__all__'].append({
+                # Prefix the actual error code with `cc-' so that it's sort
+                # of namespaced against other types of error codes.
```

Comment:
It feels dirty. I was thinking that the UI might need something like that for localization though. I'll take a look at the docs.